wrdcld/rectangle.py

```diff
@@ -47,9 +47,151 @@ def __repr__(self):
         return f"Rectangle(x={int(self.x)} y={int(self.y)} w={int(self.width)} h={int(self.height)})"
 
 
-def fill_remaining_space(
+def fill_remaining_space_horizontal(
```

Comment:
Did you remove this in the previous commit, and re-add it here?

Reply:
mayyyybe 

I had removed it, but then the filling around the words tended to make too many long narrow rectangles, so I decided to keep the old way and then only use the text filling in the text_rectangle